docs/kr.tree

```diff
@@ -323,6 +323,9 @@
             <toc-element id="maven.md" accepts-web-file-names="using-maven.html"/>
             <toc-element id="ant.md" accepts-web-file-names="using-ant.html"/>
         </toc-element>
+        <toc-element toc-title="Dokka">
+            <include from="kl" target="dokka" origin="dokka"/>
+        </toc-element>
```

Comment:
@sarahhaggarty could you please verify that I understood you correctly with the ToC?

Reply:
@IgnatBeresnev, yes this is correct!